infra.lvn_tools/extend_boot/galaxy.yml

```diff
@@ -0,0 +1,69 @@
+### REQUIRED
+# The namespace of the collection. This can be a company/brand/organization or product namespace under which all
+# content lives. May only contain alphanumeric lowercase characters and underscores. Namespaces cannot start with
+# underscores or numbers and cannot contain consecutive underscores
+namespace: infra.lvn_tools
+
+# The name of the collection. Has the same character restrictions as 'namespace'
+name: extend_boot
+
+# The version of the collection. Must be compatible with semantic versioning
+version: 1.0.0
+
+# The path to the Markdown (.md) readme file. This path is relative to the root of the collection
+readme: README.md
+
+# A list of the collection's content authors. Can be just the name or in the format 'Full Name <email> (url)
+# @nicks:irc/im.site#channel'
+authors:
+- Jordi Gil <jgil@redhat.com>
+
+
+### OPTIONAL but strongly recommended
+# A short summary description of the collection
+description: Extends the boot partition and file system of a BIOS system by taking advantage of the available PE in the adjacent LVM partition.
+
+# Either a single license or a list of licenses for content inside of a collection. Ansible Galaxy currently only
+# accepts L(SPDX,https://spdx.org/licenses/) licenses. This key is mutually exclusive with 'license_file'
+license:
+- Apache License
+
+# The path to the license file for the collection. This path is relative to the root of the collection. This key is
+# mutually exclusive with 'license'
+license_file: 'LICENSE'
```

Comment:
`license` and `license_file` and mutually exclusive. So, you should keep one and comment out or remove the other

Reply:
Dropped `license`